Wikipedia/Code/WMFLanguageCell.h

```diff
@@ -1,12 +1,14 @@
+#import "WMFTableViewCellWithCustomMinusButton.h"
 
 /// Table cell designed for displaying a language autonym & a page title in that language.
-@interface WMFLanguageCell : UITableViewCell
+@interface WMFLanguageCell : WMFTableViewCellWithCustomMinusButton
```

Comment:
I used "minus" to avoid confusion with the delete button which appears when you tap the minus button, and with the "delete" enums/methods which deal with that button... I went back and forth on it...